webapp-django/crashstats/signature/templates/signature/signature_summary.html

```diff
@@ -52,7 +52,7 @@
 
 <div class="panel">
     <header class="title">
-        <h2>Product</h2>
+        <h2>Product *</h2>
```

Comment:
You should use `<h2>Product<sup>*</sup></h2>` so you get a raised character. 
And use that too at the beginning of the text below.